.github/workflows/docker-build-static.yml

```diff
@@ -453,20 +484,26 @@ jobs:
           # list of Docker images to use as base name for tags
           images: |
             nitnelave/lldap
-          # generate Docker tags based on the following events/attributes
+            lldap/lldap
+            ghcr.io/lldap/lldap
+          # generate Docker tags based on the following events/attributes, with -alpine or -debian
+          # vX-debian / -alpine
+          # vX.Y-debian / -alpine
+          # vX.Y.Z-debian / -alpine
+          # Suffix override on latest and stable tag, stable stable-debian, latest, latest-debian
```

Comment:
We still need to have all these tags on alpine:
latest-alpine
stable-alpine
vX
vX.Y
v.X.Y.Z

Those are all tags that exist today and i don't think we should be changing that, at least not in the scope of this work. That's what i proposed in my last review, but  you can do you own take on whatever accomplishes it:

Okay, so I have an idea. I suggested the -alpine suffix, but we still need the tags without suffix. I'm thinking we add a variable to the matrix: `dobase: true` for alpine and `dobase: false` for debian. Then we duplicate tags and explicitly set suffix to empty and enable based off that:
```
            type=semver,pattern=v{{version}},suffix=,enable=${{ matrix.dobase }}
            type=semver,pattern=v{{major}}.{{minor}},suffix=,enable=${{ matrix.dobase }}
            type=semver,pattern=v{{major}},suffix=,enable=${{ matrix.dobase }}
            type=raw,value=stable,enable=${{ startsWith(github.ref, 'refs/tags/v') && matrix.dobase }},suffix=,enable=${{ matrix.dobase }}
            type=raw,value=latest,suffix=,enable=${{ github.ref == format('refs/heads/{0}', 'main') && matrix.dobase }}
```

Reply:
the `stable` and `latest` pretty much alike you suggested, just need to add for `vX` `vX.Y` `vX.Y.Z`.

Reply:
With this approach, the debian matrix will generate duplicate tags. That might be a problem, it might not; this action or the docker build might handle it just fine. I used a bool in my suggegstion to enable or disable the extra tags to avoid this - if the build is happy though, i won't oobject. One last nit here - i don';t think this generates `latest-alpine` or `stable-alpine`

Reply:
ah skipped that, and yeah `:latest` and `:stable` will duplicate. But no more because of some magic 🪄. 🤣  

Joke aside, essentially we should pass the `definition tags` to metadata generation step.